### ISSUE TITLE
design: wireframing

### ISSUE THREAD

**Issue by hacdias** [ipfs-shipyard/ipfs-desktop#669]
[WIP]

**Comment by hacdias** [comment]
Ping @ipfs-shipyard/gui for feedback!

**Comment by lidel** [comment]
I would move "Mount MFS on the File System" to postponed features.
"Way to connect to multiple backends" may also be a huge time sink if we want to get UI right, I'd say its "nice to have, but not mandatory to ship first iteration".

**Comment by alanshaw** [comment]
😱 that PR is 300 files - I'm not going to be able to review that! How can we fix?

I'm super happy we have a master issue and some issues for tasks. This is a good start. We need to make sure the other tasks get tickets with context and info on what they are before we start work on them. 

> I would move "Mount MFS on the File System" to postponed features.

+1

**Comment by hacdias** [comment]
@olizilla just removed the label `revamp` and added the related issues to milestone v1.0. Design discussion was also moved to [ipfs-gui repo](https://github.com/ipfs/ipfs-gui/issues/44).

@alanshaw @lidel about postponing "Mount MFS on the File System": do you think we should do it because the 'system' isn't ready yet or because it isn't a crucial feature even though the main feature the user will want is to add files do IPFS?

**Comment by hacdias** [comment]
@ipfs-shipyard/gui should we aim to get **signed** binaries?

**Comment by olizilla** [comment]
Yes! That is how we will honour the `1.0` release!

**Comment by olizilla** [comment]
Updated this to 0.6 release. It sends a confusing signal to users if we say Desktop is 1.0 if it's using a sub 1.0 ipfs implementation.

**Comment by daviddias** [comment]
If helpful (aka not adding much more work for little benefit) consider proposing a timeline for the outstanding tasks and creating a pre-release test phase to ask our dearest community contributors to give it a spin.
